src/test/java/org/skife/jdbi/v2/SampleBean.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2004 - 2015 Brian McCallister and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.skife.jdbi.v2;
+
+import java.math.BigDecimal;
+
+class SampleBean {
+    private Long longField;
+    protected String stringField;
+    public int intField;
+    BigDecimal bigDecimalField;
+    SampleValueType valueTypeField;
+
+    public Long getLongField() {
+        return longField;
+    }
+
+    public void setLongField(Long longField) {
+        this.longField = longField;
+    }
+
+    public String getStringField() {
+        return stringField;
+    }
+
+    protected void setStringField(String stringField) {
```

Comment:
Actually in the tests we are expecting an error from `BeanMapper`, so throwing an assertion error in the setter might trigger a false positive.

I will rename the properties to reflect intent.